### ISSUE TITLE
[Feature]: Documentation to tell newcomers to flush

### ISSUE THREAD

**Issue by jmugan** [milvus-io/milvus#16897]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

I added a bunch of records and the query didn't return the proper results. It only showed up after a certain number of records.

After a lot of trouble, I figured it out. I had to flush. This is a pretty big gotcha since it isn't in the examples for new users, at least the ones that I saw.

I had to do
```
conn = collection._get_connection()
conn.flush([collection._name])
```

### Describe the solution you'd like.

In the examples, there should be flush.

Also, flush doesn't seem to be a prominent thing in the new Python API. I had to go digging in the source code.

### Describe an alternate solution.

Or maybe flush should be called as part of `collection.load()`.

**Comment by xiaofan-luan** [comment]
Hi @jmugan ,we don't recommend you to flush before read. Actually flush too often would cause serious problems.

When you said "query didn't return the proper results", can you offer the code to reproduce the issue so we can investigate on it?
Thanks

**Comment by jmugan** [comment]
Well, hmm. If I put in around 5000 records (each a vector of size 512) and then immediately query within the same program, things work fine. But if I put in 80,000 records and then query within the same program, it seems to only pull from a subset of the vectors. Putting together the full example would require I exact the code from a blob of other code and put it back together. I'll see if I can do that in the coming days, but I'm not sure if I will be able to get to it.

**Comment by xiaofan-luan** [comment]
> Well, hmm. If I put in around 5000 records (each a vector of size 512) and then immediately query within the same program, things work fine. But if I put in 80,000 records and then query within the same program, it seems to only pull from a subset of the vectors. Putting together the full example would require I exact the code from a blob of other code and put it back together. I'll see if I can do that in the coming days, but I'm not sure if I will be able to get to it.

Guess I l know the reason.
By default we enabled a small index in growing data, which create IVF index on growing data every 32768 rows. 

We need a way to disable the small index so the result is accurate

**Comment by longjiquan** [comment]
> /assign @longjiquan could you help on this one~

I will disable small index in https://github.com/milvus-io/milvus/pull/16935. @jmugan

**Comment by longjiquan** [comment]
Close this for now. Feel free to reopen this if you have any similar problems.

**Comment by amani-acog** [comment]
I have similar issue:
I have ingested 100M records but when i saw the stats using: utility.index_building_progress(collection1_1024). I got following output
{'total_rows': 99988158, 'indexed_rows': 69198160, 'pending_index_rows': 95055457}
Why the total rows are not equal to 100M, but collection.num_entities returns 100M?
Do I need to use collection.flush() at the end of ingestion or before loading again for search?
